pipeline_dp/aggregate_params.py

```diff
@@ -246,6 +246,37 @@ def __post_init__(self):
             )
 
 
+@dataclass
+class VectorSumParams:
+    """Specifies parameters for differentially-private vector summation calculation.
+
+    Args:
+        noise_kind: The type of noise to use for the DP calculations.
+        max_partitions_contributed: A bound on the number of partitions to which one
+          unit of privacy (e.g., a user) can contribute.
+        max_contributions_per_partition: A bound on the number of times one unit of
+          privacy (e.g. a user) can contribute to a partition.
+              min_value: Lower bound on each value.
+        budget_weight: Relative weight of the privacy budget allocated to this
+          aggregation.
+        vector_norm_kind: The type of norm.
+        calculations.
+        vector_max_norm: Bound on each value of a vector.
+        vector_size: Number of coordinates in a vector.
+        partition_extractor: A function which, given an input element, will return its partition id.
+        value_extractor: A function which, given an input element, will return its value.
+    """
+    partition_extractor: Callable
+    value_extractor: Callable
+    vector_norm_kind: NormKind
+    vector_max_norm: float
+    vector_size: int
+    noise_kind: NoiseKind = NoiseKind.LAPLACE
+    max_partitions_contributed: Optional[int] = None
```

Comment:
For now these parameters are not optional for vector sum.

**Explanation**: in aggregate, there are 2 options:
to specify (`max_partitions_contributed` and `max_contributions_per_partition`)
xor to specify `max_contributions`. The 2nd option is not yet supported for VectorSum

After changing, let's keep the same order as it was before (to be consistent with other Params classes). Please ensure that the order is the same in Docstring and in code